### DISCUSSION TITLE
p.232 リスト20.13のtypo

### DISCUSSION THREAD

**Discussion by kdnakt** [budougumi0617/go_todo_app#41] [Ideas]
[後半の秘密鍵をテストする部分](https://github.com/budougumi0617/go_todo_app/blob/main/auth/jwt_test.go#L31)で、エラーメッセージにrawPubKeyを出力していますが、ここはrawPrivKeyでしょうか。

**Comment by budougumi0617** [comment]
ご指摘ありがとうございました。正誤表に反映しました。
- https://github.com/budougumi0617/go_todo_app/pull/46

// 今いただいている他の未対応の指摘もひととおり対応しだいリリースタグつけようと思います 🙇

**Reply by budougumi0617** [reply]
正誤表を更新しリリースタグをつけました。ご報告ありがとうございました。
https://github.com/budougumi0617/go_todo_app/releases/tag/v1.0.5
https://github.com/budougumi0617/go_todo_app/blob/v1.0.5/errata.md
